docs/application/native/guides/notification/notification-ex.md

```diff
@@ -0,0 +1,539 @@
+# Notification-ex
+
+
+An application can use notifications-ex to inform users about important information.
+The Notification-ex API is designed to create large and expandable notifications.
+
+The main features of the Notification-ex API include:
+
+- Creating a notification-ex item
+
+  You can [create a notification item](#create). There are 12 notification-ex item types.
+  There are five receiver-groups where you want your notification to be displayed on.
+
+- Creating a group item and add a child
+
+  You can [create a group item](#group). You can include multiple items in a group in a specific order.
+  You can create multiple groups and include them in another group.
+
+- Setting notification attributes
+
+  You can [set notification attributes](#set) such as action, style, receiver group, main type, and so on.
+
+- Creating an expanded notification
+
+  You can [create a more extended form](#expanded) of notification.
+
+- Creating a messaging style notification
+
+  You can [create a messaging style](#messaging) notification.
+
+- Creating a notification reporter handle
+
+  You need to [create a reporter handle](#reporter) to post a note.
+  While creating a handle, you can also register callbacks for events and errors.
+
+- Posting a notification
+
+  You can [post single notification and multiple notifications](#post) using `noti_ex_reporter_post()`.
+
+- Updating notification content
+
+  You can [update the content of a notification](#update) using `noti_ex_reporter_update()` function.
+
+- Deleting a notification
+
+  You can [delete notifications](#delete) when no longer needed.
+
+## Prerequisites
+
+To enable your application to use the notification-ex functionality:
+
+1. To use the Notification-ex API, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
```

Comment:
We link to API Reference files from here.
If we have expected the api reference files, please create a link, changing "Notification-ex API" to 
```
Notification-ex API (in [mobile](../../api/mobile/latest/group__NOTIFICATION__EX__MODULE.html) and [wearable](../../api/wearable/latest/group__NOTIFICATION__EX__MODULE.html) applications),
```